### PR TITLE
cookies modal

### PR DESCRIPTION
**Pull request by jota12x**
imagen
The flow is the following:

- If the user accepts the cookies, a flag will be saved in localStorage, so the next time, the modal won't appear. The **only exception** is when the user is checking the cookies policy page, as only in this page the cookies modal won't appear regardless the situation.
- If the user doesn't accept (which means, it hasn't clicked the accept button), the modal will continue to show up, which is an equivalent to say that you should accept to continue.

The alternative here is to make the user conscious of the modal without forcing it directly to click it by blocking the whole experience, however because the modal will stay, the user will have to click it at some point if he wants to continue navigating the web. 
Another alternative is to just have a footer which tells specifically that using the webpage implies accepting the cookie policy (Amazon does this), but for the search page, there is no footer properly speaking because of infinite scrolling, so I discarded that option.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,18 +2,35 @@ import React from 'react';
 import { Router, Route, Switch } from 'react-router-dom';
 import ErrorPage from 'components/pages/ErrorPage';
 import HomePage from 'components/pages/HomePage';
+import CookiesModal from 'components/molecules/CookiesModal';
 import CookieConsentPage from 'components/pages/CookieConsentPage';
 import SearchResultsPage from 'components/pages/SearchResultsPage';
 import history from 'utils/history';
 
+const acceptCookies = () => {
+  console.log('accept cookies');
```
Comment (rhezaharliman) [inline src/App.tsx]: I think we can remove this log 🤔

File: src/App.tsx [hunk 2]
```diff
@@ -2,18 +2,35 @@ import React from 'react';
...
+  localStorage.setItem('acceptCookies', 'true');
+};
+
+const hasUserAcceptedCookies = () => {
+  console.log(history.location.pathname);
```
Comment (rhezaharliman) [inline src/App.tsx]: This log too

File: src/App.tsx (hunk 1 above)
Comment (jota12x) [reply src/App.tsx]: Ops, deleted :)

File: src/App.tsx (hunk 2 above)
Comment (jota12x) [reply src/App.tsx]: Fixed :)

**Comment by rhezaharliman** [comment]
> There's one thing that comes into my mind, I think we also want to handle users that decline the cookie. Are you going to support it at the analytic's PR?

Since Juan said it'll be supported at analytics PR, this PR will be merged.

**Pull request merged**